### PR TITLE
added copy and loading animation for /me

### PR DESCRIPTION
**Pull request by mmmavis**
related to #1276

### CODE REVIEW

File: less/pages/makes.less [hunk 1]
```diff
@@ -63,3 +66,31 @@
...
+  }
+  33% {
+    width: .6rem;
+  }
+  66% {
+    width: 1.2rem;
+  }
+  100% {
+    width: 1.8rem;
+  }
```
Comment (Pomax) [inline less/pages/makes.less]: this looks like a linear animation, can we omit the 33% and 66% steps?

File: less/pages/makes.less [hunk 2]
```diff
@@ -63,3 +66,31 @@
...
+.loading-message {
+  &:after {
+    content: "...";
+    display: inline-block;
+    overflow: hidden;
+    width: 0;
+    margin-left: .5rem;
+    font-size: 2.5rem;
+    vertical-align: bottom;
+    animation: loading 1.2s infinite;
```
Comment (Pomax) [inline less/pages/makes.less]: with a `linear` in there?

File: pages/makes.jsx [hunk 3]
```diff
@@ -106,8 +106,9 @@ var MePage = React.createClass({
     if (!this.state.username) {
       pageContent = <span>Please sign in.</span>;
     } else if (this.state.loadingMakes) {
-      pageContent = <span>Loading Makes...</span>;
-    } else {
+      pageContent = <div className="loading-message">Loading makes</div>;
```
Comment (Pomax) [inline pages/makes.jsx]: What shall we call these things... "previous makes"? "older makes"? something else? /cc  @hannahkane?

Comment (mmmavis) [inline pages/makes.jsx]: do we still use the term "makes" to refer to things published using the **new tools**? if not, I think calling these "makes" is fine :eyes:

File: pages/makes.jsx [hunk 4]
```diff
@@ -120,6 +121,7 @@ var MePage = React.createClass({
       });
       pageContent = (
         <div>
+          <p className="context">In the fall of 2015, we retired Popcorn Maker and Appmaker, as well as older versions of Thimble and X-Ray Goggles. Any makes you created with these tools are still accessible below. Projects created with the new X-Ray Goggles, Thimble, or Webmaker are accessible through those respective platforms.</p>
```
Comment (Pomax) [inline pages/makes.jsx]: this isn't technically true yet...

Comment (hannahkane) [inline pages/makes.jsx]: What's not true about it?

File: pages/makes.jsx (hunk 3 above)
Comment (hannahkane) [inline pages/makes.jsx]: We're using the term "project" now. Let's say "Loading Projects..."

File: pages/makes.jsx (hunk 4 above)
Comment (hannahkane) [inline pages/makes.jsx]: Let's also replace "makes" with "projects" in this copy.

Comment (mmmavis) [inline pages/makes.jsx]: i'm going to replace "makes" with "projects" and push this live today.  we can work on tweaks later so they dont act like blocker.  :+1: :smile:

**Comment by mmmavis** [comment]
merging this now :tada:

**Pull request merged**